global_helpers/global_helpers_test.py

```diff
@@ -19,14 +19,16 @@
 
 import panther_asana_helpers as p_a_h  # pylint: disable=C0413
```

Comment:
Interesting, the `disable=C0413` is all over the place. I can address it after you merge this PR 😄